apps/condo/domains/miniapp/schema/B2BAppAccessRight.test.js

```diff
@@ -186,5 +203,327 @@ describe('B2BAppAccessRight', () => {
                 })
             }, NON_SERVICE_USER_ERROR)
         })
+
+        test('Cannot be created 2 links for a app', async () => {
```

Comment:
Please move it to `Constraints` section of tests to keep it organized (Since it more db / migrator error), then keystone hooks